bin/yaml/libraries.yaml

```
@@ -2722,6 +2722,33 @@ libraries:
       - 1.16.0
       - 1.17.0
       type: github
+  kokkos:
+      build_type: cmake
+      lib_type: static
+      make_targets:
+      - all
+      extra_cmake_arg:
+      - -DCMAKE_BUILD_TYPE=Release -DKokkos_ENABLE_CUDA=ON
```

```suggestion
      extra_cmake_arg:
      - -DCMAKE_BUILD_TYPE=Release
      - -DKokkos_ENABLE_CUDA=ON
```

I would expect it to be a list